tensorflow_lite_support/cc/test/task/vision/BUILD

```diff
@@ -173,3 +173,36 @@ cc_test_with_tflite(
         "@org_tensorflow//tensorflow/lite/kernels:builtin_ops",
     ],
 )
+cc_test_with_tflite(
+    name = "landmark_detector_test",
+    srcs = ["landmark_detector_test.cc"],
+    data = [
+        "//tensorflow_lite_support/cc/test/testdata/task/vision:test_images",
+        "//tensorflow_lite_support/cc/test/testdata/task/vision:test_models",
+    ],
+    tflite_deps = [
+        "@org_tensorflow//tensorflow/lite/core/shims:cc_shims_test_util",
+        "//tensorflow_lite_support/cc/task/core:task_api_factory",
+        "//tensorflow_lite_support/cc/task/core:tflite_engine",
```

Comment:
These two deps are not needed.

Reply:
I guess engine_ is req for checking output tensors

Reply:
and taskapifactory for CreateFromOptions test?

Reply:
Bazel recursively compiles dependencies.
```
        "//tensorflow_lite_support/cc/task/core:task_api_factory",
        "//tensorflow_lite_support/cc/task/core:tflite_engine",
```
is already in the `BUILD` file of one of your `deps`.